tests/factory_fixtures/lease.py

```diff
@@ -21,11 +21,11 @@ def _lease_attributes(unitNum, tenant, property):
 
 @pytest.fixture
 def create_lease(faker, lease_attributes, create_property, create_tenant):
-    def _create_lease(unitNum=None):
-        if not unitNum:
-            unitNum = faker.building_number()
-        tenant = create_tenant()
-        lease = LeaseModel(**lease_attributes(unitNum, tenant, create_property()))
+    def _create_lease(tenant=None, property=None, unitNum=None):
```

Comment:
Pytest has some built-in witchcraft that allows kwargs to be passed in that aren't in the fixture parameters. Discovered `tenant` and `property` only needed to be in the parameters of this inner function.

Reply:
Thanks for cleaning this up 👍